ci-operator/step-registry/baremetal/lab/upi/conf/network/baremetal-lab-upi-conf-network-commands.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+set -o errtrace
+set -o errexit
+set -o pipefail
+set -o nounset
+
+echo "Creating patch file to configure networking: ${SHARED_DIR}/network_patch_install_config.yaml"
+
```

Comment:
can we use primary and secondary in this case?

for example, have a PRIMARY_NET: ipv4|ipv6 variable (default "").

```
If dual-stack ( ipv4_enabled == true and ipv6_enabled == true) || single-stack (ipv4_enabled == true) and PRIMARY_NET="" => PRIMARY_NET=ipv4
else if PRIMARY_NET="" => PRIMARY_NET=ipv6
```

```
if PRIMARY == ipv4
  PRIMARY_NET_CLUSTER=...
  PRIMARY_NET_SERVICE
  PRIMARY_NET_MACHINE
 
if primary == ipv6 
  PRIMARY_NET_CLUSTER=...
  PRIMARY_NET_SERVICE
  PRIMARY_NET_MACHINE

if dual_stack and primary == ipv4
  SECONDARY_NET_CLUSTER=...
  SECONDARY_NET_SERVICE=...
  SECONDARY_NET_MACHINE=...


if dual_stack and primary == ipv6
  SECONDARY_NET_CLUSTER=...
  SECONDARY_NET_SERVICE=...
  SECONDARY_NET_MACHINE=...
```

the HEREDOC will be like:
```
cat > "${SHARED_DIR}/network_patch_install_config.yaml" <<EOF
networking:
  clusterNetwork:
  "${PRIMARY_NET_CLUSTER:+ - ${PRIMARY_NET_CLUSTER}}"
  "${SECONDARY_NET_CLUSTER:+ - ${PRIMARY_NET_CLUSTER}}"
...
```

bumping this back again, as i think it still needs some changes here.